src/context/UserContextProvider.tsx

```diff
@@ -168,10 +170,13 @@ export const UserContextProvider: React.FC = (props): JSX.Element => {
         handleLogin: (data): void =>
           dispatch({ type: ActionType.LOGINUSER, payload: data }),
         // Using useCallback here as these are used in useDataAPI dependency array
-        handleLogout: useCallback(
-          () => dispatch({ type: ActionType.LOGOFFUSER, payload: null }),
-          []
-        ),
+        handleLogout: useCallback(() => {
```

Comment:
Are we sure we need to use useCallback hook here?

Reply:
The `useCallback` hook was already there before I made the changes, trying without it seems to work fine I can update it to not use the hook.